### ISSUE TITLE
Unable to extend InfinityModel following README.md

### ISSUE THREAD

**Issue by scottkidder** [ember-infinity/ember-infinity#263]
When trying the approach in the docs to port Bound paramaters to the new InfinityModel extend style I was unable to get it to work, encountering an error that since the object wasn't created by the container it wasn't able to inject the global service.
I wanted to share my approach that worked for me just using an EmberObject instead of a service:

```
import EmberObject from '@ember/object';
import { get } from '@ember/object';
import { debounce } from '@ember/runloop';
import InfinityRoute from 'ember-infinity/mixins/route';
import InfinityModel from 'ember-infinity/lib/infinity-model';

export default Route.extend(InfinityRouty, {
  infinityFilters: EmberObject.create({
    search: undefined,
    type: undefined
  }),

  model() {
    const infinityFilters = this.get('infinityFilters');

    return this.infinityModel('model', {}, InfinityModel.extend({
      infinityFilters,
      buildParams() {
        const params = this._super(...arguments);
        params['search'] = get(this, 'infinityFilters.search');
        params['type'] = get(this, 'infinityFilters.type');
        return params;
      }
    }));
  },


  actions: {
     search(search) {
      this.set('infinityFilters.search', search);
      debounce(this, this.refresh, 75);
    },
    filterType(type) {
      this.set('infinityFilters.type', type);
      this.refresh();
    }
  }
});
```

**Comment by snewcomer** [comment]
Great point!  Hmmm.  Looks like we need to add some better documentation and perhaps make it a bit more flexible.  Let me see if we can get a fix in tomorrow.

**Comment by snewcomer** [comment]
Fixed in #265.  Lmk if you encounter any issues!
